Sources/Operators/CombineLatestMany.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  CombineLatestMany.swift
+//  CombineExt
+//
+//  Created by Jasdev Singh on 22/03/2020.
+//  Copyright © 2020 Combine Community. All rights reserved.
+//
+
+import Combine
+
+public extension Publisher {
+    /// A method-form of `Collection.combineLatest` in the `Publisher` namespace.
```

Comment:
I would have the same description here as the original method - the consumer reading the docs usually wouldn't care if it's a mirror of something else but more what it does IMO

Reply:
Agreed, great call. 👌🏽